shared/params/config_test.go

```diff
@@ -52,3 +55,12 @@ func TestValidatorSetDeltaFlags(t *testing.T) {
 		}
 	}
 }
+
+
+//validator params test
+func TestAttesterCommitteeSize(t *testing.T) {
```

Comment:
This has nothing to do with attester committee size. Perhaps TestCollationSize is more appropriate?